Calculations.java

```diff
@@ -0,0 +1,29 @@
+public class Calculations {
+
+    private int result;
+
+    int add(int a, int b)
+    {
+        result = a+b;
```

Comment:
Please format your code- you can do it in intellij using ctrl + alt + L shortcut I believe